src/lib/services/OutbidNotificationService.js

```diff
@@ -19,6 +19,10 @@ class OutbidNotificationService {
     return new Date(new Date().getTime() - hours * 3600 * 1000)
   }
 
+  static toParcelLink(opts) {
+    return `https://auction.decentraland.org/${opts.x}/${opts.y}`
```

Comment:
Do we need to use this code for any other URL ?, on the beta for example.

If we do, we might want to extract this to an ENV var. If not, it's not really necessary to change anything

Reply:
I was also thinking about setting that URL into an ENV variable, mostly for testing in beta and prod. Will do.